kore/src/Kore/Log/ErrorRewritesInstantiation.hs

```diff
@@ -133,15 +129,19 @@ we added to the result.
 the axiom variables from the substitution and unwrap all the 'Target's.
 -}
 checkSubstitutionCoverage
-    :: forall monadLog
+    :: forall rule monadLog
     .  MonadLog monadLog
+    => UnifyingRule rule
+    => From rule SourceLocation
+    => From rule (TermLike RewritingVariableName)
```

Comment:
It represents the rule as a term-like. This is needed if we want to print the rule in the log (see line 164). Maybe I could define a newtype over `TermLike RewritingVariableName` so that it's clearer what this instance does?